common/sgx/tlsverifier.c

```diff
@@ -56,6 +57,30 @@ static oe_result_t verify_report_user_data(
     return result;
 }
 
+// verify there is a matched claim for the public key */
+static oe_result_t verify_public_key_claim(
+    oe_claim_t* claims,
+    size_t claims_length,
+    uint8_t* pub_key_buf,
+    size_t pub_key_buf_size)
+{
+    oe_result_t result = OE_FAILURE;
+    for (int i = (int)claims_length - 1; i >= 0; i--)
+    {
+        if (oe_strcmp(claims[i].name, "public_key") == 0)
```

Comment:
Even though "public key" is used internally, it's a good idea to define it in a header file and with unique naming (e.g. "_oe_internal_public_key") so it does not conflict with other claim names defined in include/openenclave/bits/evidence.h.

Reply:
Thanks for catching that. Will do.